src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -12,18 +11,14 @@
    :padding-left      10
    :border-left-color colors/neutral-40})
 
-(defn mention-tag-wrapper
-  []
-  {:flex-direction :row
-   :align-items    :center
-   :height         22})
-
-(def mention-tag
-  {:background-color   colors/primary-50-opa-10
+(def mention-tag-wrapper
+  {:flex-direction     :row
+   :align-items        :center
+   :height             22
+   :background-color   colors/primary-50-opa-10
    :padding-horizontal 3
    :border-radius      6
-   :margin-bottom      -3
-   :height             22})
+   :transform          [{:translateY (if platform/ios? 4 6)}]})
```

Comment:
could you pls elaborate on why transform?

Reply:
@flexsurfer i explained in the PR description that margin doesn't work on nested view in android

Reply:
thanks